sky/clouds/azure.py

```diff
@@ -265,11 +265,11 @@ def check_credentials(self) -> Tuple[bool, Optional[str]]:
             output = _run_output('az account show --output=json')
         except subprocess.CalledProcessError:
             return False, (
-                'Azure CLI returned error. Run the following commands '
-                'under sky folder:'
-                # TODO(zhwu): after we publish sky to PyPI, change this to
-                # `pip install sky[azure]`
-                '\n      $ pip install .[azure]'
+                'Azure CLI returned error. Run the following commands:'
+                '\n      $ pip install skypilot[azure]  # if installed from '
+                'PyPI'
+                '\n    Or:'
```

Comment:
With the release of wheel, think `pip install .` can be removed. (only relevant for developers)

Reply:
We still include in Install docs that you can build from source. Some users do so.